ood-portal-generator/lib/ood_portal_generator/view.rb

```diff
@@ -22,6 +22,7 @@ def initialize(opts = {})
       @protocol         = @ssl ? "https://" : "http://"
       @listen_addr_port = opts.fetch(:listen_addr_port, nil)
       @servername       = opts.fetch(:servername, nil)
+      @server_aliases   = opts.fetch(:server_aliases, nil)
```

Comment:
```suggestion
      @server_aliases   = opts.fetch(:server_aliases, [])
```

If this defaults to `[]` do we need the if block in the template?

Reply:
Changed default to empty array and got rid of unnecessary condition

Reply:
Thanks!